apps/reflect/src/components/auth/AuthContainer.tsx

```diff
@@ -1,55 +1,110 @@
-import { Stack, Typography } from "@mui/material";
+import {
+  Alert,
+  Checkbox,
+  FormControlLabel,
+  Snackbar,
+  Stack,
+  Typography,
+} from "@mui/material";
 import { SiDiscord, SiTwitch } from "react-icons/si";
 import { signIn } from "next-auth/react";
 import Link from "next/link";
 import AuthLoginButton from "./AuthLoginButton";
+import { ChangeEvent, useState } from "react";
 
 const AuthContainer = () => {
+  const [acceptTos, setAcceptTos] = useState(false);
+  const [showNotAcceptedWarning, setShowNotAcceptedWarning] = useState(false);

```

Comment:
Hello, thank you very much for your contribution. Could you please add the "useState" types to the lines in this section?